### PR TITLE
Update CMSIS lib path

### PR DESCRIPTION
**Pull request by quantabase13**
The source repo change location of CMSIS lib
path, update WGET path accordingly.
Meanwhile, to align with CMSIS lib path change,
also update include path of two platforms' Makefile

### CODE REVIEW

File: platform/stm32p103/Makefile [hunk 1]
```diff
@@ -6,7 +6,8 @@ RAMSZ = 64k
 
 CFLAGS += \
     -Iplatform/stm32p103 \
-    -I$(CMSIS)/util \
+    -I$(CMSIS)\
+    -I$(CMSIS)/platform\
```
Comment (jserv) [inline platform/stm32p103/Makefile:10]: Ditto.

File: platform/f429disco/Makefile [hunk 2]
```diff
@@ -6,7 +6,8 @@ RAMSZ = 192k
 
 CFLAGS += \
     -Iplatform/f429disco \
-    -I$(CMSIS)/util \
+    -I$(CMSIS)\
+    -I$(CMSIS)/platform\
```
[the remark this answers is not in the document]
Comment (quantabase13) [reply platform/f429disco/Makefile]: done

File: platform/stm32p103/Makefile (hunk 1 above)
Comment (quantabase13) [reply platform/stm32p103/Makefile]: Done.

**Comment by jserv** [comment]
Thank @quantabase13 for contributing!

**Comment by quantabase13** [comment]
Thanks for your quick response too !

**Pull request merged**